docs/04-operation-guides/troubleshooting/api-exposure/apix-05-upgrade-sidecar-proxy.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Pods stuck in `Pending/Failed/Unknown` state after an upgrade
+---
+
+## Symptom
+
+You cannot access services or Functions using the created APIRules. Kyma Gateway refuses the connection.
+Some of your Pods are stuck in the `Pending/Failed/Unknown` state, or their sidecar proxy version differs from the installed Istio version.
+
+## Cause
+
+During the upgrade, Kyma triggers the rollout restart to the instance of resources in order to ensure full compatibility of the sidecar proxy with the newly installed Istio version. The only exceptions to this are standalone Pods (without owner reference) or Pods created by the Job controller. Pods that cannot complete the rollout restart process or the restart cannot be triggered because a Pod is a standalone resource or is created by the Job controller, get the `istio.reconciler.kyma-project.io/proxy-reset-warning` annotation with a brief explanation of the cause.
```

Comment:
I wouldn't say that the length of the sentence is the main problem. There's a mixture of clauses. There are 2 of them. The first is defining, and the other seems to be a time or conditional clause. Plus, it's complex. Putting `when` or - on second thoughts `if` - at the beginning makes the sentence easier to comprehend as the whole sentence is conditional.
Alternatively, why not split this long sentence into two shorter ones?
Have a look at these ideas:
Pods that cannot complete the rollout restart process get the `istio.reconciler.kyma-project.io/proxy-reset-warning` annotation with a brief explanation of the cause. The same happens/ The result is the same if the restart cannot be triggered because a Pod is a standalone resource or if the Job controller creates it/ a Pod.